lib/levelup.js

```diff
@@ -140,7 +140,7 @@ LevelUP.prototype.close = function (callback) {
         callback.apply(null, arguments)
     })
     this.emit('closing')
-    this.db = null
+    this.db = new DeferredLevelDOWN(this.location)
```

Comment:
this was required for tests to pass, removing `this.db = null` would result in a segfault